pom.xml

```diff
@@ -158,6 +160,14 @@
               </exclusions>
             </dependency>
 
+			<!-- This is a dependency of calcite-core and has a medium vuln. Can
+				 be removed when CALCITE-5025 is fixed -->
+			<dependency>
+			  <groupId>commons-io</groupId>
+			  <artifactId>commons-io</artifactId>
+			  <version>2.11.0</version>
```

Comment:
nit: can we move this to properties section of pom

Reply:
Nah, we won't be upgrading this but removing it when we go to calcite 1.3.0. Having it in properties section makes it easy to upgrade versions or to use the same version for related dependencies.